spec/functional/resource/user/useradd_spec.rb

```diff
@@ -77,9 +76,19 @@ def password_should_be_set
     end
   end
 
+  def try_cleanup
+    ['/home/foo', '/home/bar'].each do |f|
+      shell_out("rm", "-r", f)
+    end
+    ['cf-test'].each do |u|
+      shell_out("userdel", "-r", username)
```

Comment:
userdel isn't going to work on freebsd. The group functional tests do it this way, which is both simple and portable because it'll use the providers:

```
user(username).run_action(:remove)
```